CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+# Contributing to SKY UX
+
+We encourage contributions from all SKY UX users. To contribute, follow this step-by-step guide. For more information about our contribution guidelines, see [SKY UX Contribution Guidelines](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines).
+
+- [Contribution Process](#contribution-process)
+- [Coding Rules](#coding-rules)
+- [Commit Message Guidelines](#commit-message-guidelines)
+- [Local Development Cheatsheet](#local-development)
+
+## <a name="contribution-process"></a> Contribution Process
+
+### File an issue
+
+Before you start any work, file an issue. If an issue already exists, comment on it and mention the `@blackbaud/skyux-maintainers` team so that we know you intend to work on it. The sooner we discuss potential code changes, the better.
+
+Keep in mind that the Blackbaud design council must approve changes that impact the UI, such as new components or changes to existing components. Contributions that do not have design approval must include detailed design specifications. The design approval process can take significant time if components are complex or designs require significant changes, so account for that in your development timeline. If you need help with the design approval process, ask the SKY UX team.
+
+### Discuss the contribution with the SKY UX team
+
+After you file an issue, discuss the architecture and implementation details, requirements, and timelines with the `@blackbaud/skyux-maintainers`. We will assign a developer to your issue to help answer questions.
+
+### Developer setup
+
+1. Fork the repo into your own GitHub repo.
+1. Create a branch off of `main`. Name it after the feature to contribute, and use kebab-case (e.g. `my-new-feature`).
+1. Clone your repo locally and run `nvm use` to ensure you have the preferred version of Node.js installed. Then run `npm install` to install all required dependencies. For more information on prerequisites, see [SKY UX prerequisites](https://developer.blackbaud.com/skyux/learn/get-started/prereqs).
+
```

Comment:
Do we need a "Write the code" step here like we have on the docs site?

Reply:
I didn't think it was necessary due to it being implied by the first sentence of the "Request a preliminary review"?

Reply:
If you don't think we need a section for that step in the CONTRIBUTING.md file, I think that's fine. I just wasn't sure since we copy most of the stuff from the docs site and we include it as a separate step there.